dashboard/config/routes.rb

```diff
@@ -94,6 +95,9 @@
       end
     end
 
+    get 'docs/concepts/data-library', to: 'data_docs#index'
+    get 'docs/concepts/data-library/:data_docs_key', constraints: {data_docs_key: /#{CurriculumHelper::KEY_CHAR_RE}+/o}, to: 'data_docs#show'
+
     get 'docs/*path', to: 'curriculum_proxy#get_doc'
     get 'curriculum/*path', to: 'curriculum_proxy#get_curriculum'
```

Comment:
Will we still have the `curriculum_proxy`? Should reference guides be re-routed too?

Reply:
For reference guides it would probably be good to re-route

Reply:
Sounds good. So I think the `docs/*path` doc could be removed, as well as the `get_doc` method

Reply:
Update: Since the reference guides are now versioned, we are not going to re-map these.

Reply:
There are some old links in old scripts that point to things like [curriculum.code.org/docs/csd17-18/headers](https://curriculum.code.org/docs/csd-1718/headers/) (for example, https://studio.code.org/s/csd2-2018/lessons/4/levels/6). It sounds like y'all decided to keep this route here anyway, but that's the kind of link that I was thinking of when I kept this catch-all path

Reply:
Great, thanks! I'll clarify in a comment below.